app/src/main/java/mozilla/lockbox/presenter/RoutePresenter.kt

```diff
@@ -174,11 +180,19 @@ class RoutePresenter(
                     "This is a developer bug, fixable by adding an action to graph_main.xml"
             )
         } else {
-            val clearBackStack = src.getAction(transition)?.navOptions?.shouldLaunchSingleTop() ?: false
+
+            val action = src.getAction(transition)
```

Comment:
get the action _before_ clearing the backstack

Reply:
👍